lib/docsplit/image_extractor.rb

```diff
@@ -35,14 +35,15 @@ def convert(pdf, size, format, previous=nil)
       escaped_pdf = ESCAPE[pdf]
       FileUtils.mkdir_p(directory) unless File.exists?(directory)
       common    = "#{MEMORY_ARGS} -density #{@density} #{resize_arg(size)} #{quality_arg(format)}"
+      timeout   = 5.minutes.to_i
       if previous
         FileUtils.cp(Dir[directory_for(previous) + '/*'], directory)
-        result = `MAGICK_TMPDIR=#{tempdir} OMP_NUM_THREADS=2 gm mogrify #{common} -unsharp 0x0.5+0.75 \"#{directory}/*.#{format}\" 2>&1`.chomp
+        result = `MAGICK_TMPDIR=#{tempdir} OMP_NUM_THREADS=2 timeout #{timeout} gm mogrify #{common} -unsharp 0x0.5+0.75 \"#{directory}/*.#{format}\" 2>/dev/null`.chomp
```

Comment:
@dv This would hide all warnings, not just those that cause memory issues. Don't we benefit from the result error message in other cases?

Is it feasible to just `grep -v` the message(s) we know cause issues? Or perhaps pipe the output through `uniq` so repetitive consecutive lines are ignored?